CMakeLists.txt

```diff
@@ -85,7 +74,7 @@ endif()
 ###
 
 set(CONFIG_INSTALL_DIR "lib/cmake/${PROJECT_NAME}")
-set(INCLUDE_INSTALL_DIR "include")
+set(INCLUDE_INSTALL_DIR "include/${PROJECT_NAME}")
```

Comment:
This would have a standard customization point with proper use of `GNUInstallDirs`.